### PR TITLE
Triangle: Add setFromAttributeAndIndices().

### LINKED ISSUE #22337: Triangle.setFromPointsAndIndices buffer attribute version?
I just noticed that while geometry.vertices is gone, triangle.setFromPointsAndIndices is still there, and only used in unit test. It seems that accepting geometry.attributes.position would make more sense now, no?

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: Fixed #22337.

**Description**

Added a buffer attribute version of `setFromPointsAndIndices()`.

### CODE REVIEW

File: docs/api/en/math/Triangle.html [hunk 1]
```diff
@@ -126,6 +126,16 @@ <h3>[method:Triangle set]( [param:Vector3 a], [param:Vector3 b], [param:Vector3
...
 		</p>
 
+		<h3>[method:Triangle setFromAttributeAndIndices]( [param:BufferAttribute attribute], [param:Integer i0], [param:Integer i1], [param:Integer i2] ) [param:Triangle this]</h3>
+		<p>
+		attribute - [page:BufferAttribute] of vertex data <br />
+		i0 - [page:Integer] index <br />
+		i1 - [page:Integer] index <br />
+		i2 - [page:Integer] index<br /><br />
+
+		Sets the triangle's vectors to the vectors in the buffer attribute.
```
Comment (WestLangley) [inline docs/api/en/math/Triangle.html:136]: Something like

    Sets the triangle's vertices from the buffer attribute vertex data.

**Comment by WestLangley** [comment]
A similar method was added 9 years ago without demand. I am not sure it is used.

@makc ? What is the use case?

**Comment by mrdoob** [comment]
Hmm, I guess it's not much code anyway...

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**